app/views/favorites/index.html.erb

```diff
@@ -0,0 +1,17 @@
+
+
+
+<div class="favorite-bands">
+    <%if @favorite_artists.length >= 1%>
+        <%@favorite_artists.each do |favorite|%>
+            <ul class=<%="artist-#{favorite.artist_id}"%>>
+                <li><%=favorite.artist.name%> </li>
+                <li><%=favorite.artist.city%> </li>
+                <li><%=favorite.artist.state%> </li>
+                <li><%=favorite.artist.zipcode%> </li>
+            </ul>
+        <%end%>
+    <%else%>
+        <h3>No Favorites... Yet </h3>
+    <%end%>
```

Comment:
`@favorite_artists` returns an array from the `UserArtists` table and I have to call artists on that to access the actual artist info. 

Also, `if` the statement is in place so if the user has not favorited any artists it will display a `No Favorites... Yet`.